global.json

```diff
@@ -1,6 +1,9 @@
 {
-    "projects": ["src"],
-    "sdk": {
-      "version": "1.0.0-preview2-003154"
-    }
+  "projects": [
+    "src",
+    "test"
```

Comment:
In our 1.0 code, TestServer is in KestrelTests, not FunctionalTests. Had to add a p2p reference from FunctionalTests to KestrelTests.